packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -0,0 +1,29 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+import BigNumber from 'bignumber.js'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+/**
+ * DestroyLoanScheme DeFi Transaction
+ */
+export interface DestroyLoanScheme {
+  identifier: string // ------------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string
+  height: BigNumber // -------------------| 8 bytes unsigned integer
+}
+
+/**
+ * Composable DestroyLoanScheme, C stands for Composable.
+ * Immutable by design, bi-directional fromBuffer, toBuffer deep composer.
+ */
+export class CDestroyLoanScheme extends ComposableBuffer<DestroyLoanScheme> {
+  static OP_CODE = 0x44
+  static OP_NAME = 'OP_DEFI_TX_DESTROY_LOAN_SCHEME'
+
+  composers (dls: DestroyLoanScheme): BufferComposer[] {
+    return [
+      ComposableBuffer.varUIntUtf8BE(() => dls.identifier, v => dls.identifier = v),
+      ComposableBuffer.bigNumberUInt64(() => dls.height, v => dls.height = v)
```

Comment:
after making `height` optional you might want to change the composer also
```suggestion
      {
        fromBuffer: (buffer: SmartBuffer): void => {
          dls.height = readBigNumberUInt64(buffer)
        },
        toBuffer: (buffer: SmartBuffer): void => {
          writeBigNumberUInt64(dls.height ?? new BigNumber(0), buffer)
        }
      }
```